Mathlib/FieldTheory/Minpoly/Field.lean

```diff
@@ -118,6 +118,15 @@ theorem eq_of_irreducible_of_monic [Nontrivial B] {p : A[X]} (hp1 : Irreducible
     mul_one (minpoly A x) ▸ hq.symm ▸ Associated.mul_left _
       (associated_one_iff_isUnit.2 <| (hp1.isUnit_or_isUnit hq).resolve_left <| not_isUnit A x)
 
+theorem eq_iff_aeval_eq_zero [Nontrivial B] {p : A[X]} (hp1 : Irreducible p) (hp3 : p.Monic) :
```

Comment:
Why hp1 and hp3 as names?

Reply:
Just following the previous lemma; let me change these.